packages/components/pages/f-checkout/test/component/f-checkout-delivery.component.spec.js

```diff
@@ -79,31 +79,31 @@ describe('f-checkout "delivery" component tests', () => {
                 };
 
                 // Act
-                checkout.populateAgeVerificationForm(dob);
-                checkout.ageVerificationSubmitButton.click();
+                await checkout.populateAgeVerificationForm(dob);
+                await checkout.ageVerificationSubmitButton.click();
 
                 // Assert
-                expect(checkout.isAgeVerificationErrorDisplayed()).toBe(true);
+                await expect(await checkout.isAgeVerificationErrorDisplayed()).toBe(true);
             });
         });
     });
 });
 
 describe('f-checkout "delivery" - split notes - component tests', () => {
-    it('should open the courier and kitchen notes accordions and populate them', () => {
+    it('should open the courier and kitchen notes accordions and populate them', async () => {
         // Arrange
         checkout = new Checkout();
 
-        const args = argumentStringBuilder({ noteType: 'get-notes-config-split' });
-        checkout.withQuery('args', args);
+        const args = await argumentStringBuilder({ noteType: 'get-notes-config-split' });
+        await checkout.withQuery('args', args);
```

Comment:
This literally set a variable what is the benefit of `awaiting `it?

Reply:
Following the documentation on the wdio site, we do need to do this as otherwise it would come back as an unhandled promise rejection.